srcpkgs/tor/files/tor/run

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 [ -r conf ] && . ./conf
+ulimit -n ${MAX_OPEN_FILES:-8192}
```

Comment:
Is 8192 enough to guarantee the error won't appear again? I think double this would still be fine, file descriptors on linux are pretty cheap nowadays.

Reply:
I'd prefer to not push this too far far for now, I will see how it holds up in production. This is not HAProxy and is pretty intensive on CPU resources. I'm open to up this again if this problem will pop up again, but you need some really beefy hardware for that.

Reply:
Still there is the option to override this via conf if really needed, but I honestly doubt everyone will ever hit that.